app/views/allocations/request-places.html

```diff
@@ -0,0 +1,42 @@
+{% extends "layout.html" %}
+{% set title = 'Request places' %}
+{% set backLink = "/allocations/new-provider" %}
+{% block pageTitle %}{{ title }}{% endblock %}
+{% block breadcrumbs %}{{ macros.breadcrumbs(breadcrumbItems())}}{% endblock %}
+
+{% set requestingProviderName %}
+	{% if data['requestingProvider'] != "Find an organisation not listed above" %}
+		{{ data['requestingProvider'] }}
+	{% else %}
+		{{ data['schoolList'] }}
+	{% endif %}
+{% endset %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+  
+      <form action="/allocations/check-answers" method="post" class="form">
+        <div class="govuk-form-group">
+          <fieldset class="govuk-fieldset">
+          <legend class="govuk-fieldset__legend govuk-fieldset__legend--xl govuk-!-margin-bottom-0">
+            <h1 class="govuk-fieldset__heading govuk-!-margin-bottom-6">
+              <span class="govuk-caption-xl">{{ requestingProviderName }}</span>
+              How many places would you like to request?
+            </h1>
+            <p class="govuk-body">Say how many trainees the organisation would like to recruit - make sure this number is as accurate as possible.</p>
```

Comment:
Drop this line? The first bit just repeats the question, and the second bit seems fairly redundant?

```suggestion
```

If you need to keep something here, it should probably use the hint text and the ARIA attributes to associate it the the label? (If you use the macro this'll get set up for you)